pkg/controller/endpointslice/topologycache/topologycache.go

```diff
@@ -170,7 +178,8 @@ func (t *TopologyCache) SetNodes(nodes []*v1.Node) {
 	totalCPU := resource.Quantity{}
 
 	for _, node := range nodes {
-		if !NodeReady(node.Status) {
+		if hasExcludedLabels(node.Labels) || !NodeReady(node.Status) {
+			klog.V(2).Infof("Ignoring node %s because it is either not ready or has an excluded label", node.Name)
```

Comment:
I think is worth to split this in two if statements so we can have a more accurate log message
```suggestion
		if hasExcludedLabels(node.Labels) {
			klog.V(2).Infof("Ignoring node %s because it has an excluded label", node.Name)
			continue
		} 
		if !NodeReady(node.Status) {
			klog.V(2).Infof("Ignoring node %s because it is not ready: %v", node.Name, node.Status.Conditions)
			continue
		}
```

Reply:
Updated, thanks!